### ISSUE TITLE
Script loaded from unauthenticated sources

### ISSUE THREAD

**Issue by garethrees** [mysociety/sobanukirwa-theme#52]
![screen shot 2015-01-09 at 15 58 44](https://cloud.githubusercontent.com/assets/282788/5682514/6a7c1f5e-9818-11e4-81a4-72b055e5d894.png)

This?

```html
            <meta name="viewport" content="width=device-width, initial-scale=1.0" />
        <link href='http://fonts.googleapis.com/css?family=Source+Sans+Pro:400,600,700,900,400italic&subset=latin,latin-ext' rel='stylesheet' type='text/css'>
```

**Comment by garethrees** [comment]
Fixed in dfb3202550f950744e2c5449e0419a6b8c762859

**Comment by garethrees** [comment]
Confirmed fixed
